src/book.py

```diff
@@ -1212,6 +1212,56 @@ def detect_exchange(self, file_path: Path) -> Optional[str]:
 
         return None
 
+    def resolve_deposits(self) -> None:
+        """Matches withdrawals and deposits by referencing the related
+        withdrawal on the deposit.
+
+        A match is found when:
+            A. The coin is the same
+            B. The deposit amount is between 0.99 and 1 times the withdrawal amount.
+
+        Returns:
+            None
+        """
+        sorted_ops = tr.sort_operations(self.operations, ["utc_time"])
+
+        def is_match(withdrawal: tr.Withdrawal, deposit: tr.Deposit) -> bool:
+            return (
+                withdrawal.coin == deposit.coin
+                and withdrawal.change * decimal.Decimal(0.99)
+                <= deposit.change
+                <= withdrawal.change
+            )
+
+        withdrawal_queue: list[tr.Withdrawal] = []
+
+        for op in sorted_ops:
+            # log.debug(op.utc_time)
+            if isinstance(op, tr.Withdrawal):
+                if not misc.is_fiat(op.change):
```

Comment:
we also should check for other fiats. only `config.FIAT` hasn't to be monitored.

Reply:
What do you mean by other fiats? `misc.is_fiat` already checks if the currency is any possible currency in `Fiat`.

Reply:
Selling USD might be a taxable income for a German citizien (like selling BTC), because it's a foreign fiat currency. I think that, although this tool is for crypto taxation, we should notify about that and try to evaluate the taxes for that, too.

Reply:
what I meant by check other fiat currencies is, that we also should match deposit and withdrawals for other fiat currencies